src/openrct2/actions/TileModifyAction.cpp

```diff
@@ -23,6 +23,14 @@ TileModifyAction::TileModifyAction(
 {
 }
 
+void TileModifyAction::AcceptParameters(GameActionParameterVisitor& visitor)
+{
+    visitor.Visit(_loc);
+    visitor.Visit("setting", _setting);
+    visitor.Visit("value1", _value1);
+    visitor.Visit("value2", _value2);
```

Comment:
This doesn't expose pasteElement (which is good its much too low level how that works) but what happens if someone uses the `TileModifyType::AnyPaste` setting?

Reply:
In my tests it inserts an "empty" TileElement, that is a surface element (type 0) at height 0, surfaceStyle 0, edgeStyle 0, slope 0, ...
Now, I do not know enough about cpp to be sure about how stable that is, but I think it results from the default attribute values after calling the default constructor `TileModifyAction() = default;` (TileModifyAction.h:53).